library/client_events.js

```diff
@@ -104,7 +104,7 @@ module.exports = client => {
         let prChan = client.guilds.get(newMember.guild.id).channels.find(chan => chan.name === 'practice-room-chat')
         if (prChan == null) {
           client.log('Cannot find #practice-room-chat!')
-        } else if (restwo.permitted_channels.includes(newMember.voiceChannelID)) {
+        } else if (restwo.permitted_channels.includes(newMember.voiceChannelID) && !(newMember.selfMute && newMember.serverMute) {
```

Comment:
These are two ways to get muted, says nothing about deafened. I think you want `newMember.mute && newMember.selfDeafened`.

Reply:
Also, I think you're missing a closing `)`. Can you test this by self-hosting please?